docs/index.md

```diff
@@ -54,11 +56,11 @@ Marquez is a modular system and has been designed as a highly scalable, highly e
   <img src="./assets/images/design.png">
 </figure>
 
-To ease adoption and enable a diverse set of data processing applications to build metadata collection as a core requirement into their design, Marquez provides language-specific clients that implement the [Metadata API](./openapi.html). As part of our initial release, we have provided support for [Java](https://github.com/MarquezProject/marquez-java) and [Python](https://github.com/MarquezProject/marquez-python).
+To ease adoption and enable a diverse set of data processing applications to build metadata collection as a core requirement into their design, Marquez provides language-specific clients that implement the [Metadata API](./openapi.html). As part of our initial release, we have provided support for [Java](https://github.com/MarquezProject/marquez/tree/main/clients/java) and [Python](https://github.com/MarquezProject/marquez/tree/main/clients/python).
```

Comment:
More curious than anything - how do these differ from the OpenLineage clients?

Reply:
Yeah, the Marquez clients allow for reading metadata, searching, and querying the lineage graph. While the OpenLineage clients are used for collecting lineage events.